trajectory_evaluation.ipynb

```diff
@@ -318,17 +307,33 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Total error of ucb-sdb-ios-1 for each trip \n",
-    "print(spatial_errors['ios']['ucb-sdb-ios-1']['total_error'])"
+    "ifig, ax_array = plt.subplots(nrows=2,ncols=3,figsize=(16,8), sharex=False, sharey=False)\n",
+    "timeline_list = [\"train_bus_ebike_mtv_ucb\", \"car_scooter_brex_san_jose\", \"unimodal_trip_car_bike_mtv_la\"]\n",
+    "for i, tl in enumerate(timeline_list):\n",
+    "    spatial_errors_df.query(\"timeline == @tl & phone_os == 'android'\").boxplot(ax = ax_array[0][i], column=[\"total_error\"], by=[\"quality\"], showbox=False, whis=\"range\")\n",
+    "    ax_array[0][i].set_title(tl)\n",
+    "    spatial_errors_df.query(\"timeline == @tl & phone_os == 'ios'\").boxplot(ax = ax_array[1][i], column=[\"total_error\"], by=[\"quality\"], showbox=False, whis=\"range\")\n",
```

Comment:
I didn't show the box in the power case because there were only three points. But because this has significantly more points, I think we can remove the showbox and whis parameters from the end. But I can also do that before I put the plots into the paper...